tests/config/test_data.config

```diff
@@ -26,6 +26,8 @@ params {
 
                 all_sites_fas                                  = "${test_data_dir}/genomics/sarscov2/genome/alignment/all_sites.fas"
                 informative_sites_fas                          = "${test_data_dir}/genomics/sarscov2/genome/alignment/informative_sites.fas"
+
+                contigs_genome_alignment_maf_gz                = "${test_data_dir}/genomics/sarscov2/genome/alignment/last/contigs.genome.maf.gz"
```

Comment:
```suggestion
                contigs_genome_maf_gz                = "${test_data_dir}/genomics/sarscov2/genome/alignment/last/contigs.genome.maf.gz"
```

Reply:
Name here should be the same as file name so need to change one or the other to match.